### PR TITLE
Add Flare

### PR DESCRIPTION
**Pull request by tmagicturtle**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -36,6 +36,62 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Ice Shard', target);
+		},
+		onEffectiveness(typeMod, target, type) {
+			if (type === 'Water') return 0;
+		},
+		effect: {
+			noCopy: true,
+			onStart(pokemon) {
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: smack down, use that.

File: data/mods/ssb/random-teams.ts [hunk 2]
```diff
@@ -34,6 +34,12 @@ export const ssbSets: SSBSets = {
 	// Nature needs to be a valid nature with the first letter capitalized ex: Modest
 	*/
 	// Please keep sets organized alphabetically based on staff member name!
+	Flare: {
+		species: 'Weavile', ability: 'Permafrost Armor', item: 'Life Orb', gender: 'M',
```
Comment (Volco) [inline data/mods/ssb/random-teams.ts]: ``FlareToday at 5:00 PM
Ah minor fix on my Pokémon please. I noticed in the code it says the gender is "m" but I stated that it should be genderless``
Gender should be N

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -37,6 +37,7 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 	},
 	*/
 	// Please keep sets organized alphabetically based on staff member name!
+
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]: remove this blank line imo

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -76,6 +77,34 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Fire",
 	},
+      
+
+	// Flare
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
	// Flare
```

**Pull request merged**